examples/petsc/bp3.h

```diff
@@ -31,7 +31,7 @@ CEED_QFUNCTION(SetupDiff)(void *ctx, const CeedInt Q,
   CeedScalar *qd = out[0], *true_soln = out[1], *rhs = out[2];
 
   // Quadrature Point Loop
-  CeedPragmaOMP(simd)
+  #pragma omp simd
```

Comment:
Hey @jedbrown why using this version, different from `CeedPragmaSIMD`?

Reply:
My mistake; pushed an amended commit.